cpmonitor/models.py

```diff
@@ -199,48 +199,54 @@ class Meta:
 
     # 3. Umsetzungsstand
 
-    class ExecutionAssessment(models.IntegerChoices):
-        UNKNOWN = 0, "unbekannt (grau)"
-        AHEAD = 1, "vor dem Plan (grün)"
-        AS_PLANNED = 2, "im Plan (grün)"
-        DELAYED = 3, "zurückgestellt / verzögert (orange)"
-        INSUFFICIENT = 4, "nicht ausreichend / in Teilen gescheitert (rot)"
-        FAILED = 5, "gescheitert (rot)"
-        MISSING = 6, "fehlt im Plan (rot)"
-
-    execution_assessment = models.IntegerField(
-        "Bewertung des Umsetzungsstandes",
-        choices=ExecutionAssessment.choices,
-        default=ExecutionAssessment.UNKNOWN,
+    class ExecutionStatus(models.IntegerChoices):
```

Comment:
text values are not according to specification in [WeChange](https://rocketchat.wechange.de/group/lz_localmonitoring_kernteam/thread/yX5FHpvnWubo7mTnp):

- should be "unbekannt" instead of "unbekannt / ungeplant". "ungeplant" is ambiguous compared to "fehlt" and "fehlt" should be together with "verzögert", so that it's red for the signal effect
- should be "in Arbeit", not "geplant / in Arbeit". Anything that is not obviously delayed is "in Arbeit". I would call the key IN_PROGRESS, not AS_PLANNED

Reply:
I must admit that I have quite some issues with this. But I think the difference came about, because there was some time-overlap of my work and Philipps.

If you use the texts for the front-end, go ahead and change them in your PR as you like them.

Reply:
Would you mind specifying the issues you're having (if it can be done in two lines)? Just to take them into consideration.